packages/nodes-base/nodes/Airtop/actions/file/upload.operation.ts

```diff
@@ -34,7 +34,8 @@ export const description: INodeProperties[] = [
 		type: 'string',
 		default: '',
 		required: true,
-		description: 'Name for the file',
+		description:
+			'Name for the file to upload. Files loaded into the session should have <b>unique names</b>.',
```

Comment:
maybe `'Name for the file to upload. For a session, all files loaded should have <b>unique names</b>.',`